README.md

```diff
@@ -2072,33 +2074,104 @@ interface ICreateX {
+
+You can broadcast the transaction using either [`ethers.js`](https://docs.ethers.org/v6/) or [`cast`](https://book.getfoundry.sh/reference/cli/cast):
+
+#### [`ethers.js`](https://docs.ethers.org/v6/)
+
+It is recommended to install [`pnpm`](https://pnpm.io) through the `npm` package manager, which comes bundled with [Node.js](https://nodejs.org/en) when you install it on your system. It is recommended to use a Node.js version `>= 18.0.0`.
+
+Once you have `npm` installed, you can run the following both to install and upgrade `pnpm`:
+
+```console
+npm install -g pnpm
+```
+
+After having installed `pnpm`, simply run:
+
+```console
+git clone https://github.com/pcaversaccio/createx.git
+cd createx
+pnpm install
+```
+
+Now configure your target chain in the [`hardhat.config.ts`](./hardhat.config.ts) file with the `networks` and `etherscan` properties, or use one of the preconfigured network configurations. After you have made sure locally that the `gasLimit` of 3 million works on your target chain, you can invoke:
+
+```console
+npx hardhat run --no-compile --network <NETWORK_NAME> scripts/deploy.ts
+```
 
-TBD (section on pre-signed transactions; we will offer multiple pre-signed transactions with different `gasLimit` levels; will be added after the feedback phase)
+The [`deploy.ts`](./scripts/deploy.ts) script ensures that [`CreateX`](./src/CreateX.sol) is automatically verified if you have configured the `etherscan` property accordingly. The current script broadcasts the _default_ pre-signed transaction, which has a gas limit of 3,000,000 gas. If you want to use a different pre-signed transaction, you must change the import of the pre-signed transaction in the [`deploy.ts`](./scripts/deploy.ts) script.
+
+#### [`cast`](https://book.getfoundry.sh/reference/cli/cast)
+
+It is recommended to install [Foundry](https://github.com/foundry-rs/foundry) via:
+
+```console
+curl -L https://foundry.paradigm.xyz | bash
+foundryup
+```
+
+If you want or need to build [Foundry](https://github.com/foundry-rs/foundry) from source code, you can simply invoke the following command to install the full toolkit:
+
+```console
+cargo install --git https://github.com/foundry-rs/foundry --profile local --force --locked forge cast chisel anvil
+```
+
+To broadcast a pre-signed transaction, you can invoke:
+
+```console
+# $TX is the pre-signed transaction.
+# $RPC_URL is the RPC URL of the target chain to which you want to deploy.
+cast publish $TX --rpc-url $RPC_URL
+```
+
+You must verify the [`CreateX`](./src/CreateX.sol) contract separately, as specified in the next section.
+
+> [!IMPORTANT]
+> After deployment, please open a pull request that updates the [`deployments.json`](./deployments/deployments.json) file and the [`CreateX` Deployments](#createx-deployments) section with the new deployment so that other users can easily know that it has been deployed.
```

Comment:
I like to have it in the `README` as well since it doesn't force you to leave the page (I loved it when you had all multicall deployments in the `README` and I didn't have to click on a separate URL...). If it uses too much space we can easily do `<summary>` `<details>` expandable content as I have done in other places already. The maintenance is easy, as I have done this for `Create2Deployer` for 2 years now :); it's one entry in the `JSON` and one bullet point entry in the `README`.

Reply:
Good call on hiding it behind a details block. Ok, I'm good with that and will mark this resolved